graylog2-shared/src/main/java/org/graylog2/shared/security/ldap/LdapEntry.java

```diff
@@ -41,6 +54,13 @@ public String put(String key, String value) {
         return attributes.put(key.toLowerCase(), value);
     }
 
+    public void addGroups(Collection<String> groups) {
```

Comment:
Could we replace this method by `setGroups()` to reduce mutable fields in this class?